homeassistant/components/melcloud/climate.py

```diff
@@ -136,6 +150,7 @@ def __init__(self, device: MelCloudDevice, ata_device: AtaDevice) -> None:
 
         self._attr_name = device.name
         self._attr_unique_id = f"{self.api.device.serial}-{self.api.device.mac}"
+        self._attr_hvac_action = None
```

Comment:
That is the default.

Reply:
Should I not override default class member with attribute member?